.github/workflows/ci-docker-satellites.yml

```diff
@@ -60,11 +60,25 @@ jobs:
       EARTHLY_ORG: "earthly-technologies"
     secrets: inherit
 
-  docker-tests-no-qemu-quick:
+  docker-tests-no-qemu-quick1:
```

Comment:
Is `quick` still a suitable name? :stuck_out_tongue:

Reply:
hahah right? we do have a `-quick`, `-normal`, and `-slow` targets, maybe these should all just be renamed to `tests-no-qemu-group{1,2,3,...10}`.

the `-quick` target was simply the first one in our tests file so I think everyone just added new tests to it, to the point it was no longer quick.

Reply:
Yeah, I wonder if we should group them by domain. Or just split them with `group` as you suggested. By domain may be a challenge as there are so many distinct groups under `./tests`.